sound/soc/sof/sof-priv.h

```diff
@@ -409,6 +409,8 @@ struct sof_ipc_pm_ops {
 	int (*set_core_state)(struct snd_sof_dev *sdev, int core_idx, bool on);
 };
 
+#define SOF_FW_PARSE_MANIFEST_PRE_BOOT	BIT(1)
```

Comment:
maybe add a comment on what this really means, I couldn't figure out what the concept was and which flags are changed,.